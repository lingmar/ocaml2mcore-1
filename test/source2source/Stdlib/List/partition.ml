let rec print_list = function
  | [] -> ()
  | x :: xs ->
    print_int x;
    print_list xs

let l = List.init 10 (fun x -> x)

let l, u = List.partition (fun x -> x > 4) l

let _ =
  print_list l;
  print_endline "";
  print_list u
