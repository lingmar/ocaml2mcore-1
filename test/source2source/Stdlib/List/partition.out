56789
01234